lib/src/participant/local.dart

```diff
@@ -250,6 +243,30 @@ class LocalParticipant extends Participant<LocalTrackPublication> {
 
     await track.start();
 
+    String? primaryCodecMime;
+    for (var codec in trackInfo.codecs) {
+      primaryCodecMime ??= codec.mimeType;
+    }
+
+    if (primaryCodecMime != null) {
```

Comment:
should only update if track is video

Reply:
flutter SDK uses a separate function `publishVideoTrack` to publish track, so there should be no need to check `track is video`

Reply:
ah that makes sense. sorry I missed that.